wpilib/wpilib/doublesolenoid.py

```diff
@@ -5,13 +5,13 @@
 # must be accompanied by the FIRST BSD license file in the root directory of
 # the project.
 #----------------------------------------------------------------------------
+from .sendablebuilder import SendableBuilder
 
 import hal
-import enum
+from enum import IntEnum
```

Comment:
I'm not a fan of this change. This just means more lines we need to change when we inevitably decide to go and change everything to plain enums.

Reply:
yeah and I realize it's more pythonic to import enum and then show that you're getting IntEnum from enum anyway. I'm changing back

Reply:
Although I'm going to leave this for a different PR anyway